.github/workflows/release.yml

```diff
@@ -39,6 +37,21 @@ jobs:
           dart: true
           rust: true
 
+      - name: Download artifacts
+        id: artifacts
+        uses: actions/download-artifact@f023be2c48cc18debc3bacd34cb396e0295e2869 # v2.1.0
+        with:
+          path: ${{ runner.temp }}/artifacts/
+
+      - name: Copy artifacts
+        working-directory: ${{ steps.artifacts.outputs.download-path }}
+        run: |
+          cp -R ${{ needs.build-ios-libs.outputs.artifact-dir-base }}-*/* ${{ github.workspace }}/${{ env.FLUTTER_WORKSPACE }}/ios
```

Comment:
Maybe we should also `mkdir -p` the ios libs path.

Through I guess it's currently not needed as other files are in there, too :thinking: